rsts/deployment/sandbox.rst

```diff
@@ -190,3 +190,7 @@ Flyte configuration on your remote cluster.
     kubectl port-forward --address 0.0.0.0 svc/envoy 30081:80 -n projectcontour
 
 #. Open console http://localhost:30081/console.
+
+#. In order to interact with your Flyte instance using ``flytectl``, initialise your configuration to point to this host ::
+
+    flytectl config init --host='http://localhost:30081'
```

Comment:
make it `flytectl config init --host='localhost:30081'`

Reply:
I'm embarrassed! Mindless copy-pasting the wrong command. fixing